jobs/async-upload/samples/sample_job_s3_to_oci.yaml

```
@@ -10,6 +10,9 @@ metadata:
     app.kubernetes.io/managed-by: kubectl
     component: model-registry-job
     modelregistry.kubeflow.org/job-type: async-upload
+    modelregistry.kubeflow.org/registered-model-id: "1"
+    modelregistry.kubeflow.org/model-version-id: "v3"
+    modelregistry.kubeflow.org/model-artifact-id: "6"
```

nit, as MV.id are still numbers

```suggestion
    modelregistry.kubeflow.org/model-version-id: "2"
    modelregistry.kubeflow.org/model-artifact-id: "3"
```

---

oops correct, referring to registered versions the actual not the `version` field itself.